edit.ascx

```diff
@@ -0,0 +1,129 @@
+                    <nav class="navbar navbar-expand-lg">
+                       
+                            
+                            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                                <dnn:MENU MenuStyle="assets/menus/topmenu" runat="server" ExcludeNodes="Admin,Host"></dnn:MENU>
+                            </div>
+                        
+                    </nav>
+                    </div>
+                    </div>
+                    </div>
+                </header>
+            </div>
+
+                <!-- Page content-->
+                <main class="skin-main content-wrap">
+               
+                
+                <div class="panes">
+                    <div class="container">
+                        <div class="row">
+                            <div class="col col-12">
+                                <div class="pane content-pane" id="contentpane" runat="server" />
+                            </div>
+                        </div>
+                    
+                    </div>
+                </div>
+                
+                </main>
+                <footer class="skin-footer bg-blue-super-dark skew-top">
+                    <div class="container py-5">
+                        <div class="row gx-5">
+                            <div class="col">
+                                <div class="pane footer_pane_1 col" id="footer_pane_1" runat="server">
+                                </div>
+                            </div>
+                            <div class="col">
+                                <div class="pane footer_pane_2 col" id="footer_pane_2" runat="server">
+                                </div>
+                            </div>
+                            <div class="col">
+                                <div class="pane footer_pane_3 col" id="footer_pane_3" runat="server">
+                                </div>
+                            </div>
+                            <div class="col">
+                                <div class="pane footer_pane_4 col" id="footer_pane_4" runat="server">
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="pane footer_pane_full_width pb-0" id="footer_pane_full_width" runat="server">
+                    </div>
+                    <div class="footer-bottom">
+                        <div class="container pt-2">
+                            <div class="row">
+                            <!-- #include file="Includes/footer-bottom.ascx" -->
+                            </div>
+                        </div>
+                    </div>
+                </footer>
+                
+               
+            </div>
+ 
+        <!-- Bootstrap core JS-->
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js"></script>
```

Comment:
Awesome! My only concern is maintaining bootstrap js. When loaded from a CDN we don't have dependabot pickup that we are using it. I would prefer if we can pull this from package management (package.json).

Reply:
Ok, np

Reply:
@Timo-Breumelhof it appears this is still being loaded from CDN instead of from the node package.  Were you planning to update this?

Reply:
@david-poindexter yes, I thought I did, I'll check

Reply:
OK i seems I missed the line in edit.ascx.. I'll correct that.

Reply:
Okay, it looks like this is now resolved.  Thanks!